stacks/api/app.module.ts

```diff
@@ -21,7 +21,7 @@ import {DashboardModule} from "@spica-server/dashboard/dashboard.module";
       defaults: [CREATED_AT, UPDATED_AT]
     }),
     PreferenceModule,
-    BucketModule,
+    BucketModule.create(process.env.ENABLE_BUCKET_HOOKS),
```

Comment:
Shall we rename it forRoot to follow the convention? 
Also, I think it would be good if we have a BucketOptions which has a property to toggle hooks. What do you think?

We can make it work like below;

```typescript
BucketModule.forRoot({
 hooks: !!process.env.ENABLE_BUCKET_HOOKS
})
```